data/abilities.ts

```diff
@@ -955,8 +994,10 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 				this.damage(pokemon.baseMaxhp / 8, pokemon, pokemon, this.dex.species.get(speciesid));
 			}
 		},
-		isBreakable: true,
-		isPermanent: true,
+		flags: {
+			failroleplay: 1, noreceiver: 1, noentrain: 1, notrace: 1, failskillswap: 1, cantsuppress: 1,
+			breakable: 1, notransform: 1,
+		},
```

Comment:
```suggestion
		flags: {
			failroleplay: 1, noreceiver: 1, noentrain: 1, notrace: 1, failskillswap: 1, cantsuppress: 1,
			breakable: 1, notransform: 1,
		},
```
i think we try to keep these on one line, afaik we do this for moves?

Reply:
We do generally, but sometimes the linter forces it onto 2 lines because it's too long. Some moves have the same problem (e.g. Me First).